abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/arrow-cpp-feedstock/pr38/8d2d038
```

Comment:
Note that builds are failing due to https://anaconda.slack.com/archives/C03J3NQPS0Z/p1707155680293859.